pkg/metrics/policystatemetrics/policystatemetrics.go

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package policystatemetrics
+
+import (
+	"context"
+
+	"github.com/cilium/tetragon/api/v1/tetragon"
+	"github.com/cilium/tetragon/pkg/logger"
+	"github.com/cilium/tetragon/pkg/metrics/consts"
+	"github.com/cilium/tetragon/pkg/observer"
+	"github.com/cilium/tetragon/pkg/sensors"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type TracingPolicyState int
+
+const (
+	LoadErrorState TracingPolicyState = iota
+	ErrorState
+	DisabledState
+	EnabledState
+)
+
+func (s TracingPolicyState) String() string {
+	switch s {
+	case LoadErrorState:
+		return "load_error"
+	case ErrorState:
+		return "error"
+	case DisabledState:
+		return "disabled"
+	case EnabledState:
+		return "enabled"
+	default:
+		return "unknown"
+	}
+}
+
+// GetTracingPolicyState get the implicit state for a TracingPolicy when loaded
+// by the agent. The state is here determined by the order of the if statements.
+//
+// Note: it's maybe not the best place to store this function and the
+// TracingPolicyState but this needs to be shared for tetra tracingpolicy list
+// command human output to use.
+func GetTracingPolicyState(policyStatus *tetragon.TracingPolicyStatus) TracingPolicyState {
+	if policyStatus.LoadError != "" {
+		return LoadErrorState
+	}
+	if policyStatus.Error != "" {
+		return ErrorState
+	}
+	if policyStatus.Enabled {
+		return EnabledState
+	}
+	return DisabledState
+}
```

Comment:
See @tixxdz @kkourt @lambdanis I added this so that this logic can be technically moved at some point outside the metrics package and maybe be part of the TracingPolicy logic. I could also reuse this in `tetra` and simplify the text output by printing the same `state` that we have in the metrics. 

It actually simplified the code and made it easier to test so I think this is what I was searching for initially.